purchase_analytic_global/models/purchase.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2014-2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    account_analytic_id = fields.Many2one(
+        "account.analytic.account",
+        string="Analytic Account",
```

Comment:
Odoo doesn't like same string in 2 fields, you must rename it, because Analytic Account it is already used on account_analytic_ids field.